core/modules/book/book.theme.inc

```diff
@@ -36,6 +36,7 @@ function template_preprocess_book_all_books_block(&$variables) {
  * @see book-navigation.tpl.php
  */
 function template_preprocess_book_navigation(&$variables) {
+  $navigation_options = config_get('book.settings','book_navigation_options');
```

Comment:
Can we move this `config_get` inside the check for `if ($book_link['mlid']) {` so we aren't loading it when it's not needed?